table/iterator.go

```diff
@@ -352,6 +352,15 @@ func (itr *Iterator) Value() (ret y.ValueStruct) {
 	return
 }
 
+// CopyValue copies the current value and returns it as decoded
+// ValueStruct.
+func (itr *Iterator) CopyValue() (ret y.ValueStruct) {
```

Comment:
Can you please call this `ValueCopy` so that it's similar to the `item.ValueCopy()` method?